toolchains/cpu.spack.gyselalibxx_env/prepare.sh

```diff
@@ -79,7 +88,11 @@ then
     exit 1
 fi
 
-. ${CURRENT_DIR}/spack-0.23.1/share/spack/setup-env.sh
+export SPACK_PATH=${CURRENT_DIR}/spack-0.23.1/
+export SPACK_USER_CONFIG_PATH="\${SPACK_PATH}/user_config"
+export SPACK_SYSTEM_CONFIG_PATH="\${SPACK_PATH}/sys_config"
+export SPACK_USER_CACHE_PATH="\${SPACK_PATH}/user_cache"
+. \${SPACK_PATH}/share/spack/setup-env.sh
```

Comment:
Why do we need to escape the `$` symbol ?

Reply:
@tpadioleau good question! It's because I want the resulting environment.sh to look like:
```
export SPACK_PATH=/....path_to_gyselalibxx/toolchains/cpu.spack.gyselalibxx_env/spack-0.23.1/
export SPACK_USER_CONFIG_PATH="${SPACK_PATH}/user_config"
export SPACK_SYSTEM_CONFIG_PATH="${SPACK_PATH}/sys_config"
export SPACK_USER_CACHE_PATH="${SPACK_PATH}/user_cache"
. ${SPACK_PATH}/share/spack/setup-env.sh
```
so that it's easy to adapt if user decides to use another spack installation, or to move the exsting one, ...

Reply:
Ok thanks, looks good to me.